src/NATS.Client.Core/Commands/CommandWriter.cs

```diff
@@ -725,7 +777,9 @@ private async ValueTask PongStateMachineAsync(bool lockHeld, CancellationToken c
         }
     }
 
+#if NET6_0_OR_GREATER
```

Comment:
Should we rather use `!NETSTANDARD` here?

Reply:
yea maybe, less to update when we move .NET targets up